spec/integration/layout_periphery_boundingbox_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe 'when configuring the header and footer background colors' do
+
+  class PeripheryExtensionTest < ::Asciidoctor::Pdf::Converter
+
+    def build_pdf_options doc, theme
+      # assert the theme will call us for header and footer
+      theme.header_background_color = "009246"
+      theme.header_border_width = 0
+      theme.footer_background_color = "CE2B37"
+      theme.footer_border_width = 0
+      theme.header_height = 160
+      theme.footer_height = 160
+      theme.page_margin = [ 160, 48, 160, 48]
+
+      super doc, theme
+    end
+
+  end
+
+  it "should draw the configured color across the whole periphery region", :integration => true do |test|
```

Comment:
Please use the modern Hash syntax, `integration: true`. Also, I don't think the block need an argument since I don't see it used anywhere.

Reply:
If we're going to mark tests as integration, then they don't need to be in a separate folder.